<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkOutDialog</class>
 <widget class="QDialog" name="checkOutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font: 13pt &quot;Arial Rounded MT Bold&quot;;
background-color:rgba(1,25,53,200);
color:rgba(208,233,255);
bordercolor:rgba(1,25,53,90);
border-radius:3px;
}
QLabel
{
font: 11pt &quot;Arial&quot;;
color:rgb(1, 25, 53);
}
QLineEdit
{
border-color:rgba(255,255,255,200);
font: 11pt &quot;Arial&quot;;
border-color:rgba(1,25,53,50);
           border-radius:3px;
         background-color:rgba(255,255,255);
      color:rgba(1,25,53);
}
QRadioButton
{
font: 11pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(1, 25, 53);
}
QTableWidget
{
font: 11pt &quot;Arial&quot;;
color:rgb(1, 25, 53);
}
</string>
  </property>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Display time here</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Null</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>MemberName</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="levelLabel">
      <property name="text">
       <string>Null</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="goodsNumberEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Commodity Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addItemButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>341</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QLabel" name="finalCostLabel">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Discount: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="currrentPointLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="IncrementLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Final Total Cost: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current points:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Point Increment: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="finalDiscountlabel">
      <property name="text">
       <string>1.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>184</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Original Total Cost: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="displayTCostLabel">
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>550</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>timeLabel</zorder>
  <zorder>line</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>goodsNumberEdit</zorder>
  <zorder>addItemButton</zorder>
  <zorder>tableWidget</zorder>
  <zorder>gridLayoutWidget_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>quantityEdit</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
